build.md

```diff
@@ -57,6 +57,20 @@ Install the build dependencies.
   * Build the binary.
   * Build the docker image with the binary.
 
+* To build multi-arch images, you need to have `buildx` installed. You can follow these [instructions](https://github.com/docker/buildx#installing).
+Supported architectures are: amd64, arm64, armv7, ppc64le. 
+  * NDM multi-arch images are built using QEMU mode(Note: This can take a lot of time since it runs in an emulated environment). If you're on linux, run :
```

Comment:
```suggestion
  * NDM multi-arch images are built using QEMU mode (**NOTE**: This can take a lot of time since it runs in an emulated environment). If you're on linux, run :
```